language/diem-framework/src/natives/mod.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod account;
+pub mod signature;
+
+use move_core_types::{account_address::AccountAddress, identifier::Identifier};
+use move_vm_runtime::native_functions::NativeFunction;
+
+pub fn all_natives(
+    addr: AccountAddress,
```

Comment:
Small thing: maybe make it clear what this address is for, something like `diem_framework_address` or something